recipes/flytekit/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "flytekit" %}
+{% set version = "0.26.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flytekit-{{ version }}.tar.gz
+  sha256: 83bf9ed9d014b0683f49a668113266f4a2f394d4870cd733a9b73b438fe8ef23
+
+build:
+  skip: true  # [py2k]
```

Comment:
```suggestion
  skip: true  # [py2k or win]
```

Reply:
You could add notes under the line like
#Skipping Windows for now
#xref: https://github.com/conda-forge/protobuf-feedstock/issues/133